connector_prestashop/account.py

```diff
@@ -102,14 +95,17 @@ def comment(self, record):
     @mapping
     @only_create
     def invoice_lines(self, record):
-        slip_details = record.get('associations', {}).get('order_slip_details', []).get('order_slip_detail', [])
+        slip_details = record.get(
+            'associations', {}
+        ).get('order_slip_details', []).get(
+            self.backend_record.get_version_ps_key('order_slip_detail'), [])
         if isinstance(slip_details, dict):
             slip_details = [slip_details]
         lines = []
         fpos_id = self.from_sale_order(record)['fiscal_position']
```

Comment:
The field name is correct: https://github.com/odoo/odoo/blob/75a196972e9e1056c8ab9c299ddfc8e2eaafd812/addons/sale/sale.py#L248

Are you seeing to 9.0 version?